notifique/src/main/java/com/nathanrassi/notifique/NotifiqueListView.kt

```diff
@@ -129,27 +130,31 @@ internal class NotifiqueListView(
     val queryProvider = { limit: Long, offset: Long ->
       val searchText = searchText
       if (searchText == null) {
-        notifiqueQueries.notifiques(limit, offset)
+        savedNotifiqueQueries.notifiques(limit, offset)
       } else {
-        notifiqueQueries.notifiquesSearch(searchText, limit, offset)
+        savedNotifiqueQueries.notifiquesSearch(searchText, limit, offset)
       }
     }
     val countQueryProvider = {
       val searchText = searchText
       if (searchText == null) {
-        notifiqueQueries.count()
+        savedNotifiqueQueries.count()
       } else {
-        notifiqueQueries.countSearch(searchText)
+        savedNotifiqueQueries.countSearch(searchText)
       }
     }
-    val transacter = notifiqueQueries
+    val transacter = savedNotifiqueQueries
+    val notificationContentIntents = mapOf<String, PendingIntent>()
     pagingSourceFactory = InvalidatingPagingSourceFactory(
-      QueryPagingSourceFactory(
```

Comment:
I couldn't make this work totally right. Data set changes caused imperfect animations.